src/main/java/org/jboss/logmanager/MDCAdapter.java

```diff
@@ -0,0 +1,76 @@
+package org.jboss.logmanager;
+
+import java.util.Map;
+
+public interface MDCAdapter {
```

Comment:
I would prefer `MDCProvider` rather than `MDCAdapter` since nothing is really being adapted.

Reply:
For sure, changed to provider